corehq/apps/dump_reload/couch/load.py

```diff
@@ -53,7 +53,10 @@ def _get_db_for_doc_type(self, doc_type):
             if couch_db is None:
                 raise DocumentClassNotFound('No Document class with name "{}" could be found.'.format(doc_type))
             callback = LoaderCallback(self._success_counter, self.stdout)
-            db = IterDB(couch_db, new_edits=False, callback=callback)
+            chunksize = 100
+            if doc_type == "Application":
```

Comment:
actually I think you might need to consider also `LinkedApplication` and `RemoteApp`, and possibly the `-Deleted` versions of those.